components/tag/TagCard.vue

```diff
@@ -32,13 +40,13 @@ function go(evt: MouseEvent | KeyboardEvent) {
 
 <template>
   <div
-    block p4 hover:bg-active flex justify-between cursor-pointer
+    block p4 hover:bg-active flex justify-between cursor-pointer flex-gap-2
```

Comment:
This gap also improves the layout of the hashtag page in Explore tab for very long hashtag names.

![Screenshot from 2024-02-25 20-21-16](https://github.com/elk-zone/elk/assets/1425259/0919f2b3-949a-485d-bf06-bb5566e7bfe4)

Reply:
I think for long tags the start should be top aligned

Reply:
I'd like the tag to always be aligned with the text below. I proposed having the star to the right, but I see now that it isn't a good idea. Maybe we should have the star always at the top left, and then the tag and the text below to the right of it. Or the star could be at the right of the graph, kind of like the Follow button in the notification cards for people following you

Reply:
OK, I tried another layout by making the follow button an independent block and placing it on the left. This ensures both texts are aligned and more robust with a long hashtag.

For Hovercard, `items-center` looks lined up better than the top. For Hashtag, both look fine. 

## Hovercard
![Screenshot from 2024-02-25 21-19-41](https://github.com/elk-zone/elk/assets/1425259/4da00f69-ceea-4ec2-b95a-f37f83763e8a)

### align center (`items-center`)
![Screenshot from 2024-02-25 21-19-06](https://github.com/elk-zone/elk/assets/1425259/8062e18c-ab55-48ef-8805-f4175db46069)
![Screenshot from 2024-02-25 21-10-25](https://github.com/elk-zone/elk/assets/1425259/b37d8652-a77a-4f4b-bb37-060d9dc22275)

### align top (`items-start`)
![Screenshot from 2024-02-25 21-18-56](https://github.com/elk-zone/elk/assets/1425259/3a2add8d-5370-4ddf-8734-50e36148ef21)
![Screenshot from 2024-02-25 21-11-57](https://github.com/elk-zone/elk/assets/1425259/0f447de4-fbc1-4bb4-984b-10a8256f9ad1)

## Hashtag page

![Screenshot from 2024-02-25 21-44-05](https://github.com/elk-zone/elk/assets/1425259/3d5e8443-f3fe-4296-9945-9317fdb0b83e)
![Screenshot from 2024-02-25 21-43-37](https://github.com/elk-zone/elk/assets/1425259/901e17a6-fe78-4210-8ae1-22a4f5a00fb5)